wazuh/wazuh
# Fix Windows graceful shutdown

|Related issue|
|---|
|#12091|



## Description

This PR aims to fix the graceful shutdown problem in Windows

Closes #12091 

## Tests
To test the behavior, the following changes have been performed:
- Disable all the modules except for FIM
- Disable FIM
- Disable FIM DB Synchronization



- Compilation without warnings in every supported platform


- Memory tests for Linux
  - [ ] Dr. Memory
- Memory tests for Windows
  - [ ] Dr. Memory
